### PR TITLE
AudioLowPassFilter Support

### PR DESCRIPTION
**Pull request by vale7z4387ruwrejmhsr**
so the reason why i didn't make a pull request for audiolowpass and reverbfilter is becausei was just kinda "lazy"

### CODE REVIEW

File: AssetRipperCore/Classes/AudioLowPassFilter/AudioLowPassFilter.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+			if (HasCutoffFrequency(reader.Version))
+			{
+				CutoffFrequency = reader.ReadSingle();
+			}
+			LowpassResonanceQ = reader.ReadSingle();
+			lowpassLevelCustomCurve.Read(reader);
+		}
+
+		public static bool HasCutoffFrequency(UnityVersion version) => version.IsLessEqual(5, 3);
```
Comment (ds5678) [inline AssetRipperCore/Classes/AudioLowPassFilter/AudioLowPassFilter.cs:30]: This should be `version.IsLess(5, 2);`. Where are you getting your information?

File: AssetRipperCore/Classes/AudioLowPassFilter/AudioLowPassFilter.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+			node.Add("m_LowpassResonanceQ", LowpassResonanceQ);
+			node.Add("m_lowpassLevelCustomCurve", GetlowpassLevelCustomCurve().ExportYAML(container));
+			return node;
+		}
+
+		private AnimationCurveTpl<Float> GetlowpassLevelCustomCurve()
+		{
+			lowpassLevelCustomCurve = new AnimationCurveTpl<Float>(1.0f, 0.0f, 1.0f / 3.0f);
+			return lowpassLevelCustomCurve;
+		}
```
Comment (ds5678) [inline AssetRipperCore/Classes/AudioLowPassFilter/AudioLowPassFilter.cs:46]: Delete this method

File: AssetRipperCore/Classes/AudioLowPassFilter/AudioLowPassFilter.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+		public static bool HasCutoffFrequency(UnityVersion version) => version.IsLessEqual(5, 3);
+
+		protected override YAMLMappingNode ExportYAMLRoot(IExportContainer container)
+		{
+			YAMLMappingNode node = base.ExportYAMLRoot(container);
+			node.AddSerializedVersion(1);
+			node.Add("m_CutoffFrequency", HasCutoffFrequency(container.Version));
+			node.Add("m_LowpassResonanceQ", LowpassResonanceQ);
+			node.Add("m_lowpassLevelCustomCurve", GetlowpassLevelCustomCurve().ExportYAML(container));
```
Comment (ds5678) [inline AssetRipperCore/Classes/AudioLowPassFilter/AudioLowPassFilter.cs:38]: This should be `node.Add("lowpassLevelCustomCurve", lowpassLevelCustomCurve.ExportYAML(container));`

File: AssetRipperCore/Classes/AudioLowPassFilter/AudioLowPassFilter.cs [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+			lowpassLevelCustomCurve.Read(reader);
+		}
+
+		public static bool HasCutoffFrequency(UnityVersion version) => version.IsLessEqual(5, 3);
+
+		protected override YAMLMappingNode ExportYAMLRoot(IExportContainer container)
+		{
+			YAMLMappingNode node = base.ExportYAMLRoot(container);
+			node.AddSerializedVersion(1);
+			node.Add("m_CutoffFrequency", HasCutoffFrequency(container.Version));
```
Comment (ds5678) [inline AssetRipperCore/Classes/AudioLowPassFilter/AudioLowPassFilter.cs:36]: Replace
```cs
node.Add("m_CutoffFrequency", HasCutoffFrequency(container.Version));
```
With
```cs
if (HasCutoffFrequency(container.ExportVersion))
{
    node.Add("m_CutoffFrequency", CutoffFrequency);
}
```

**Comment by vale7z4387ruwrejmhsr** [comment]
i should've known there was something wrong

**Pull request merged**